### PR TITLE
update go mod for volcano.sh/apis

### PR DESCRIPTION
**Pull request by huone1**
Signed-off-by: huone1 <huwanxing@huawei.com>

### CODE REVIEW

**Review by shinytang6 — APPROVED** [review]
/lgtm

**Comment by huone1** [comment]
optimize the volcano-mimitor sheill script, delete the repeat data

File: hack/generate-yaml.sh [hunk 1]
```diff
@@ -100,6 +100,11 @@ echo "Generating volcano yaml file into ${DEPLOYMENT_FILE}"
 if [[ -f ${DEPLOYMENT_FILE} ]];then
     rm ${DEPLOYMENT_FILE}
 fi
+
+if [[ -f ${MONITOR_DEPLOYMENT_YAML_FILENAME} ]];then
```
Comment (Thor-wl) [inline hack/generate-yaml.sh:104]: Why introduce monitor part in generate-yaml.sh

Comment (huone1) [reply hack/generate-yaml.sh:104]: I don't know Why introduce monitor part in generate-yaml.sh, this change is just a optimization， If Not， the operater in volcano-monitoring-latest.yaml is append and has the repeat data ; you have a look at it currently , you can find it

**Review by shinytang6 — APPROVED** [review]
/lgtm

**Review by william-wang — APPROVED** [review]
/lgtm

**Pull request merged**